keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -254,33 +258,29 @@ Metadata:
   - cosi.io/finalizer [3]
 spec:
   bucketClassName: [4]
-  supportedProtocols: [5]
-  releasePolicy: {"Delete", "Retain"} [6]
-  bucketRef: [7]
+	bucketRef: [5]
     name:
     namespace:
-  secretName: [8]
-  accessMode: {"rw", "ro"} [9]
+  secretRef: [6]
```

Comment:
do you want to call out `name:` and `namespace:` as done for `bucketRef`?

Reply:
Not _really_ as they're already core pieces of k8s.